sklearn/ensemble/_base.py

```diff
@@ -85,13 +85,13 @@ class BaseEnsemble(MetaEstimatorMixin, BaseEstimator, metaclass=ABCMeta):
 
     Parameters
     ----------
-    base_estimator : object, optional (default=None)
+    base_estimator : object or None, default=None
         The base estimator from which the ensemble is built.
 
-    n_estimators : integer
+    n_estimators : int, default=10
         The number of estimators in the ensemble.
 
-    estimator_params : list of strings
+    estimator_params : list of strings or tuple of strings, default=tuple()
```

Comment:
You can remove the change here. list of strings is fine. We initialize with an empty tuple to avoid having a mutable as a default argument.